metadata.go

```diff
@@ -80,12 +83,69 @@ var debianLicenseText = map[string]string{
 `,
 }
 
+var githubRegexp = regexp.MustCompile(`github\.com/([^/]+/[^/]+)`)
+
+func findGitHubRepo(gopkg string) (repo string, err error) {
+	if strings.HasPrefix(gopkg, "github.com/") {
+		repo = strings.TrimPrefix(gopkg, "github.com/")
+		return
```

Comment:
Here and elsewhere, please don’t use naked returns:

> Naked returns are okay if the function is a handful of lines. Once it's a medium sized function, be explicit with your return values. Corollary: it's not worth it to name result parameters just because it enables you to use naked returns. Clarity of docs is always more important than saving a line or two in your function.

(from https://github.com/golang/go/wiki/CodeReviewComments#named-result-parameters)